applications/tari_mining_node/src/stratum/stratum_miner/miner.rs

```diff
@@ -132,8 +132,8 @@ impl StratumMiner {
                     if difficulty >= target_difficulty {
                         let block_header: BlockHeader = BlockHeader::try_from(hasher.into_header()).unwrap();
                         info!(
-                            "Miner found block header {} with difficulty {:?}",
-                            block_header, difficulty,
+                            "Miner found block header {} with nonce {} and difficulty {:?}",
+                            block_header, solver.current_nonce, difficulty
```

Comment:
```suggestion
                            "Miner found solution with nonce: {} and difficulty: {:?} and block header: {}",
                            solver.current_nonce, difficulty, block_header
```

Reply:
Thanks @hansieodendaal.

Will modify this shortly

Would not recommend adding the header since it will be a fair block of text for each solution, the height may be a better idea, or maybe a secondary debug!() message with it in?

Reply:
Included the hash in the info message and the full block_header in a secondary debug message.